db/migrate/20160407183719_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :role, :string, null: false, default: "member"
```

Comment:
It's fine that you added `null: false` here, but it's made pretty redundant by `default: "member"`... Because you're assigning a default, there *will never be* a time when it can be null